ts/private/ts_project.bzl

```diff
@@ -148,9 +148,10 @@ See https://github.com/aspect-build/rules_ts/issues/361 for more details.
     for a in ctx.files.assets:
         a_path = _lib.relative_to_package(a.short_path, ctx)
         a_out = _lib.to_out_path(a_path, ctx.attr.out_dir, ctx.attr.root_dir)
-        asset = ctx.actions.declare_file(a_out)
-        copy_file_action(ctx, a, asset)
-        assets_outs.append(asset)
+        if a_path != a_out:
```

Comment:
I think we might still have to copy if `a.is_source` just so it gets copied to bindir? I think the bug is when the file is already in the bindir..?

Reply:
Added a.is_source test also

Reply:
Did any test fail without that? Want to try adding one?

Reply:
How can I do this ?

Reply:
I might try to do it later, I think you've tested the bug you're fixing so this lgtm 👍